### PR TITLE
Provide German translations

### PR DESCRIPTION
**Pull request by kghbln**

### CODE REVIEW

**Comment by kghbln** [comment]
@JohnFLewis Another thank you. :)

File: i18n/de.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"@metadata": {
+		"authors": [ "Kghbln" ]
+	},
+	"rottenlinks": "Status externer Links",
+	"rottenlinks-desc": "Ermöglicht die Anzeige des [[Special:RottenLinks|Status]] der auf dem Wiki genutzten externen Links",
+	"rottenlinks-table-external": "Externer Link",
+	"rottenlinks-table-response": "HTTP-Antwort",
+	"rottenlinks-table-usage": "Seitenanzahl",
+	"rottenlinks-showbad": "Lediglich untergegangene Links anzeigen",
```
Comment (paladox) [inline i18n/de.json]: Syntax error, need to remove the "," please

**Comment by kghbln** [comment]
Yeah, nice find. Done.

**Comment by paladox** [comment]
Thanks!

**Pull request merged**